lib/versioning/pvp/index.ts

```diff
@@ -0,0 +1,80 @@
+import { regEx } from '../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../loose/generic';
```

Comment:
Looks like it needs import fixes
```suggestion
import { GenericVersion, GenericVersioningApi } from '../generic';
```